ARrehab/ARrehab/Minigames/Pseudo Code.swift

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
What does this file do? it seems incomplete. Also the title suggests pseudo code but this seems like Swift.